### PR TITLE
BaSP-TG-63: Shared modal component

### PR DESCRIPTION
**Pull request by MaxiG-dev**
-Create modal messages component

### CODE REVIEW

File: src/Components/Shared/Modal/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  cursor: pointer;
+}
+
+.content button {
+  background-color: rgb(27, 195, 83);
+  color: black;
+}
+.content button:first-child {
+  background-color: rgb(195, 27, 27);
+}
```
Comment (CristianBucci) [inline src/Components/Shared/Modal/modal.module.css:45]: Please set the properties order like this:
- Position 
- Display.
- Flexbox (flex-direction, align-self, etc).
- Box-model: width, height, margin, padding, top, left, etc.
- Background y border.
- Text (font-size, text-shadow, etc).
- Other rules

File: src/Components/Shared/Modal/Modal.message.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            <img src={`${process.env.PUBLIC_URL}/assets/images/close.svg`} alt="Close icon" />
+          </button>
+        </span>
+        <div className={styles.content}>
+          <span>{modalContent}</span>
+        </div>
+      </div>
+    </div>
+  ) : (
+    ''
```
Comment (nicolasblasco) [inline src/Components/Shared/Modal/Modal.message.js:20]: We should return null instead of an empty string

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Modal/Modal.message.js (hunk 2 above)
Comment (ignacioaristo) [reply src/Components/Shared/Modal/Modal.message.js]: If we only want to show it when show is equal to true we could use the `&&` operator.
Like this:
`return show && (......)
`
With that, we avoid use the ternary in the wrong way.

**Review by ignacioaristo — APPROVED** [review]

File: src/Components/Shared/Modal/modal.module.css (hunk 1 above)
Comment (MaxiG-dev) [reply src/Components/Shared/Modal/modal.module.css:45]: Done!

File: src/Components/Shared/Modal/Modal.message.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        </span>
+        <div className={styles.content}>
+          <span>{modalContent}</span>
+        </div>
+      </div>
+    </div>
+  ) : null;
+};
+
+export default ModalMessage;
```
Comment (nicolasblasco) [inline src/Components/Shared/Modal/Modal.message.js:22]: We should rename the component to Modal.js

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by CristianBucci — APPROVED** [review]

File: src/Components/Shared/Modal/Modal.message.js (hunk 3 above)
Comment (MaxiG-dev) [reply src/Components/Shared/Modal/Modal.message.js]: Done!

File: src/Components/Shared/Modal/Modal.message.js (hunk 2 above)
Comment (MaxiG-dev) [reply src/Components/Shared/Modal/Modal.message.js]: Done!

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**